From 7b7e3ce7c062e4892faa88894366c2e9bfffa6f6 Mon Sep 17 00:00:00 2001
From: Louis Marelle <louis.marelle@latmos.ipsl.fr>
Date: Wed, 23 Feb 2022 16:47:15 +0100
Subject: [PATCH 03/50] Remove wrf_message output from photolysis driver

In photolysis driver, replace the call to wrf_message by a call to
wrf_debug (15) for the message indicating the driver has been called, in
agreement with other WRF drivers.
---
 chem/photolysis_driver.F | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chem/photolysis_driver.F b/chem/photolysis_driver.F
index a1a4be1d..5603777d 100755
--- a/chem/photolysis_driver.F
+++ b/chem/photolysis_driver.F
@@ -149,7 +149,7 @@
    IF (config_flags%phot_opt .eq. 0) return
 
    write(dbg_msg,*) 'photolysis_driver: called for domain ',id
-   CALL wrf_message( trim(dbg_msg) )
+   CALL wrf_debug(15, trim(dbg_msg))
 !
 ! select photolysis option
 !
-- 
2.31.1

